### ISSUE TITLE
Model Incomplete

### ISSUE THREAD

**Issue by hayscoder** [khoanguyen0791/ally-api-client#1]
Hello! I noticed that the model is not all complete. For example, the "com.celexus.conniption.model.accounts.Instrument" class does not have the putcall element.

I couldn't find the "tradeking-model" as a github repository. Is there a reason why the source code isn't out there?

I would be willing to contribute to your project and update the model with the new elements.

Is that possible?

**Comment by mw66** [comment]
@hayscoder not sure if this is still maintained (@ecigar13 ?), I just find an issue:

https://github.com/ecigar13/ally-api-client/issues/3

and fixed in:

https://github.com/ecigar13/ally-api-client/pull/2

If you plan to work on it, maybe you can just create a new repo, and I will contribute to it.

**Comment by hayscoder** [comment]
Yeah, doesn't seem to be maintained. I just switched to Alpaca. Much simpler and better service. Ally actually overcharges on it's federal fees, fyi.

**Comment by mw66** [comment]
Thanks for the info.

BTW, How much Ally actually overcharges on it's federal fees compare with Alpaca? just want a rough estimate, to see if I want to continue using Ally.

**Comment by mw66** [comment]
Are you talking about regulatory fees? I found this:

https://alpaca.markets/support/regulatory-fees/
https://www.ally.com/go/invest/commissions-and-fees/

looks like they are the same?

**Comment by hayscoder** [comment]
Yes, the regulatory fees. It is pretty minimal but I just don't appreciate
how ally handles it. They always round up on each trade. As, you can see
the fees are a fraction of a share.

So, take a drastic example. Say that you have 10 sale trades for 1 share of
stock in each trade. Ally will essentially charge you 2 cents for every
trade (20 cents total). I am not sure exactly the calc that Alpaca is using
but they essentially charge you once you accumulate up to a cent of fees.
So, they will not charge you 20 cents in this instance but might charge you
1 or 2 cents.

Again, yes tiny amounts. However, I wonder how much money ally is making
behind the scenes in a year across all their clients because of this hidden
margin.

From an API perspective, the alpaca api is way better too.

The biggest downside to alpaca right now in my opinion is the type of
accounts. I can have a Roth IRA using the ally api ... But I can't do that
today with Alpaca. But if you are doing just a normal person account, then
that doesn't matter.

Just my 20 cents ....
